apps/jobbrowser/src/jobbrowser/tests.py

```diff
@@ -676,6 +676,12 @@ def test_app(self):
       'doc_url': 'http://url.com/query_plan?query_id=8a46a8865624698f:b80b211500000000'}.items():
       assert_equal(response.get(key), value)
 
+  def test_parse_job_timestamp(self):
```

Comment:
@g3rg0 Can we shift this test in `query_api_tests.py`?

Reply:
I'm closing this PR because @bjornalm solved the problem in #3398 too, and the tests are in the right place there :)

Reply:
Hi @g3rg0 really sorry for the duplicate work. Vacation brain here and there was a urgent request to have this fixed. I updated my PR to include your check for missing millisecond dot.

Reply:
@bjornalm No worries, it's fine anyway :) The point is that the problem is solved.